### PR TITLE
F #src 3778 benchmarking

### PR DESCRIPTION
**Pull request by ethanfowler**

### CODE REVIEW

File: sr_system_info/src/sr_system_info/system_info.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import rospy
+import rospkg
+import subprocess
+import yaml
+
+
+class SystemInfo(object):
+
+    def __init__(self):
+        self.rospack = rospkg.RosPack()
```
Comment (mikramarc) [inline sr_system_info/src/sr_system_info/system_info.py]: private variables with leading underscore

File: sr_system_info/src/sr_system_info/system_info.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+    def collect(self):
+        self.values = {}
```
Comment (mikramarc) [inline sr_system_info/src/sr_system_info/system_info.py]: all member variables should be defined in the init method, its our convention apparently

File: sr_system_info/src/sr_system_info/system_info.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        self.values["ros_root_path"] = rospkg.get_ros_root()
+        self.values["ros_package_paths"] = rospkg.get_ros_package_path().split(':')
+        self.values["src_package_paths"] = [i for i in self.values["ros_package_paths"] if "src" in i]
+        self.survey_packages()
+        self.survey_source_repos()
+        self.survey_system()
+        # rospy.loginfo(self.yaml())
```
Comment (mikramarc) [inline sr_system_info/src/sr_system_info/system_info.py]: please remove commented out code

**Review by mikramarc — CHANGES_REQUESTED** [review]

File: sr_system_info/src/sr_system_info/system_info.py (hunk 3 above)
Comment (ethanfowler) [reply sr_system_info/src/sr_system_info/system_info.py]: Fixed in https://github.com/shadow-robot/sr_utils/pull/6/commits/321bb546d91af88f074b8164506d2429c7b1d232

File: sr_system_info/src/sr_system_info/system_info.py (hunk 2 above)
Comment (ethanfowler) [reply sr_system_info/src/sr_system_info/system_info.py]: Fixed in https://github.com/shadow-robot/sr_utils/pull/6/commits/321bb546d91af88f074b8164506d2429c7b1d232

File: sr_system_info/src/sr_system_info/system_info.py (hunk 1 above)
Comment (ethanfowler) [reply sr_system_info/src/sr_system_info/system_info.py]: Fixed in https://github.com/shadow-robot/sr_utils/pull/6/commits/321bb546d91af88f074b8164506d2429c7b1d232

**Review by mikramarc — APPROVED** [review]

**Pull request merged**